packages/core/i18n/core-i18n-server-internal/src/init_translations.ts

```diff
@@ -11,10 +11,5 @@ import { i18n, i18nLoader } from '@kbn/i18n';
 export const initTranslations = async (locale: string, translationFiles: string[]) => {
   i18nLoader.registerTranslationFiles(translationFiles);
   const translations = await i18nLoader.getTranslationsByLocale(locale);
-  i18n.init(
-    Object.freeze({
-      locale,
-      ...translations,
-    })
-  );
+  i18n.init(Object.freeze({ ...translations }));
```

Comment:
NIT TODO assess whether we can remove the `Object.freeze()`

Reply:
Opened an issue to tackle it later on: https://github.com/elastic/kibana/issues/180810